CMakeLists.txt

```diff
@@ -490,64 +490,10 @@ IF(WITH_TEST_TRACE_PLUGIN AND NOT CMAKE_BUILD_TYPE MATCHES "Debug")
           Set WITH_TEST_TRACE_PLUGIN to OFF or WITH_DEBUG to ON.")
 ENDIF()
 
-<<<<<<< HEAD
-# Set DBUG_OFF and other optional release-only flags for non-debug project types
-FOREACH(BUILD_TYPE RELEASE RELWITHDEBINFO MINSIZEREL)
-  FOREACH(LANG C CXX)
-    SET(CMAKE_${LANG}_FLAGS_${BUILD_TYPE} 
-     "${CMAKE_${LANG}_FLAGS_${BUILD_TYPE}} -DDBUG_OFF -DNDEBUG")
```

Comment:
There is "-DNDEBUG" added by us here. Is it Ok to remove it?

Reply:
I thought a lot a lot and came to the conclusion it is already populated by default. Please, consider next reasonings:

-   In the following code it suggess it suggests it is a default value:
    <https://github.com/mysql/mysql-server/blob/a9b0c712de3509d8d08d3ba385d41a4df6348775/CMakeLists.txt#L809-L822>

-   MySql has removed other references to other flags as `-g`.

-   My local `CMakeCache.txt` contains the following values

    ```
    CMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG
    ...
    CMAKE_C_FLAGS_RELEASE:STRING=-O3 -DNDEBUG
    ...
    ```

On the other way, I haven't found any reference to the default values. It might something which depends on the platform. I will double check with the author of the change to see if there is a reason to keep it.

Reply:
> In the following code it suggess it suggests it is a default value

In general it's possible that this comment refers to these lines which set defaults.